tests/base/converters/test_dataframe_to_mds.py

```diff
@@ -17,7 +17,11 @@
 from streaming.base.converters import dataframeToMDS
 
 MY_PREFIX = 'train'
-MY_BUCKET = 'mosaicml-composer-tests'
+MY_BUCKET = {
+    'gs://': 'mosaicml-composer-tests',
```

Comment:
Please use some random names for buckets, for example, `testing-bucket`. The same applies to below two lines.

Reply:
Done.